bibliopixel/drivers/SPI/APA102.py

```diff
@@ -1,24 +1,43 @@
 from ... import gamma
 from .. channel_order import ChannelOrder
+from ... import data_maker
 from . base import SPIBase
 
 
 class APA102(SPIBase):
-    """Driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
+    """Driver for APA102/SK9822 based LED strips on devices like the Raspberry Pi and BeagleBone"""
 
     def __init__(self, num, gamma=gamma.APA102, **kwargs):
         super().__init__(num, gamma=gamma, **kwargs)
+        maker = kwargs.get('maker', data_maker.MAKER)
```

Comment:
Perhaps rather DriverBase should store the `maker` as a member variable?  I can imagine that we'll do this again...